css/revops.css

```diff
@@ -144,17 +163,35 @@ p {
   margin-bottom: 1.5em;
 }
 
+.h2-wrapper {
```

Comment:
Naming: h2 is more general than where this wrapper is actually used. I would use something like `revops-section__heading-wrapper` for a BEM name: a wrapper of headings inside revops-sections. That's the context for using this as far as we know.

If we had a more general name for the type of page (rather than "revops") that would be even better, but we don't currently have plans to replicate this exact look on other pages.

C.f. https://getbem.com/naming/

Reply:
fixed